src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -113,6 +113,25 @@ export interface CreateApiKeyOptions {
     apiKeyTemplateId?: string;
 }
 
+export interface DuplicateApiKeyOptions {
+    /**
+     * The display name for the API key.
+     *
+     * **Example:** PushApiKey
+     */
+    displayName?: string;
+    /**
+     * A brief description of the API key.
+     *
+     * **Example:** API key used for managing sources.
+     */
+    description?: string;
+    /**
+     * The lifetime duration of the API key (iso8601 format).
+     */
+    lifetimeDuration: string;
```

Comment:
I don't think the examples are necessary for the `displayName` and `description` as they are simple strings. It would be more worthwhile to add one for the `lifetimeDuration`. I think you should use `@example` instead of `**Example**`